data-model/digitalobjects/0.2.0/digital-specimens/schema/occurrences.json

```diff
@@ -0,0 +1,262 @@
+      "examples": [
+        "https://orcid.org/0000-0002-1825-0097"
+      ]
+    },
+    "???:eventName": {
+      "type": "string",
+      "comment": "Not sure what this field means"
+    },
+    "dwc:fieldNumber": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/fieldNumber",
+      "examples": [
+        "RV Sol 87-03-08"
+      ]
+    },
+    "dwc:recordNumber": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/recordNumber",
+      "examples": [
+        "OPP 7101"
+      ]
+    },
+    "dwc:eventDate": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/eventDate",
+      "examples": [
+        "1963-03-08T14:07-0600"
+      ]
+    },
+    "dwc:verbatimEventDate": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/verbatimEventDate",
+      "examples": [
+        "Marzo 2002"
+      ]
+    },
+    "dwc:year": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/year",
+      "maximum": 9999,
+      "examples": [
+        2008
+      ]
+    },
+    "dwc:month": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/month",
+      "maximum": 12,
+      "minimum": 1,
+      "examples": [
+        12
+      ]
+    },
+    "dwc:day": {
+      "type": "integer",
+      "description": "https://rs.tdwg.org/dwc/terms/day",
+      "maximum": 31,
+      "minimum": 1,
+      "examples": [
+        29
+      ]
+    },
+    "dwc:habitat": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/habitat",
+      "examples": [
+        "savanna"
+      ],
+      "$comment": "Is this necessary for specimens"
+    },
+    "???:protocolDescription": {
+      "type": "string",
+      "comment": "Not sure what this field means"
+    },
+    "dwc:sampleSizeValue": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/sampleSizeValue",
+      "examples": [
+        "5"
+      ]
+    },
+    "dwc:sampleSizeUnit": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/sampleSizeUnit",
+      "examples": [
+        "meters"
+      ]
+    },
+    "dwc:samplingProtocol": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/samplingProtocol",
+      "examples": [
+        "collected directly from the wild"
+      ]
+    },
+    "???:eventEffort": {
+      "type": "string",
+      "comment": "Not sure what this field means"
+    },
+    "dwc:fieldNotes": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/fieldNotes",
+      "examples": [
+        "Notes available in the Grinnell-Miller Library"
+      ]
+    },
+    "dwc:eventRemarks": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/eventRemarks",
+      "examples": [
+        "After the recent rains the river is nearly at flood stage"
+      ]
+    },
+    "???:collectorName": {
```

Comment:
this is usually captured by recordedBy?

Reply:
I wouldn't say recorder and collector are necessarily the same. However, just as with typeDesignatedBy, we need to rework the agents to a generic object which includes the relationship to the object.